lib/sensors/temperature.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "sensors.hpp"
+
+#include <cstdint>
+#include <cstdio>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+const std::uint8_t CTRL          = 0x04;
+const std::uint8_t TEMPERATURE_1 = 0x38;
```

Comment:
`kTemperatureAddress1` unclear on this, @SnickeyX thoughts

Reply:
So we don't want any device addresses, since the device addresses of these will simply be the mux's. Instead the constructor of these sensors on the mux should take in the `mux_adderess` as a parameter and call its reads/writes with that.